core/src/main/java/io/grpc/Server.java

```diff
@@ -38,4 +41,57 @@
  * application code or interceptors.
  */
 @ThreadSafe
-public abstract class Server {}
+public abstract class Server {
```

Comment:
Because it needs to be managed, `ManagedServer` is what I thought we were going to call it. That is why we didn't call the Channel ChannelService, because ServerService would be a really weird name.